src/components/PostList/PostList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Post from '../Post/Post';
+import CommentList from '../CommentList/CommentList';
+
+const PostList = ({ list }) => (
+  <>
+    {list.map(post => (
+      <div className="Post" key={post.id}>
+        <Post {...post} />
+        <CommentList {...post} />
```

Comment:
pass here just comments
```suggestion
        <CommentList comments={comments} />
```